src/View/CrudView.php

```diff
@@ -6,6 +6,32 @@
 
 class CrudView extends View
 {
+    public $layout = 'CrudView.default';
+
+    public function initialize(array $config = [])
```

Comment:
This won't work - the BootstrapUI readme is wrong (should not include `array $config = []`, not how `View::initialize()` is done). The README was fixed but I guess it was only done for the coming 0.4 release, sorry.